### DISCUSSION TITLE
Deployment status for the "issue_comment" trigger

### DISCUSSION THREAD

**Discussion by MiguelRipoll23** [community/community#9032] [Actions]
Triggering a production deployment from a comment is the most convenient way when approving a pull request but unfortunately even when deploying the PR's branch the status is not linked to the deployment due to the _issue_comment_ trigger not providing a build status.

Using the GitHub API to add and update the deployment status is a little bit cumbersome.

**Comment by ethomson** [comment]
Hi @MiguelRipoll23 - thanks for the feedback - can you give me a few more details?  I don't feel like I have a complete picture yet, and I think that you're doing some very cool but also advanced stuff.  I'd love to see your workflow, would you be willing to share it?

**Reply by MiguelRipoll23** [reply]
Hi @ethomson! Sure, [this is for my example my PR](https://github.com/MiguelRipoll23/tusestimaciones-web/pull/1), it was deployed both to staging and production but only the status from the staging deployment was linked. The “pull_request” trigger can be used to link a deployment status but the “issue_comment” trigger doesn’t share the same behavior. When the comment “/approve” was posted the production deploy workflow was started.

[This is the workflow file, it uses a public reusable workflow.](https://github.com/MiguelRipoll23/tusestimaciones-web/blob/main/.github/workflows/deploy-production.yml)

Thanks for your response!
